app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    com, first_file, second_file = command.split()
```

Comment:
I'd suggest using the whole word if you aren't completely sure it's necessary. It only saves you few characters anyway